gobblin-core/src/main/java/gobblin/source/extractor/utils/InputStreamCSVReader.java

```diff
@@ -49,7 +49,7 @@ public InputStreamCSVReader(BufferedReader input) {
 
   public InputStreamCSVReader(String input) {
     this(new InputStreamReader(new ByteArrayInputStream(input.getBytes()),
-        Charset.forName(ConfigurationKeys.DEFAULT_CHARSET_ENCODING)));
+        Charset.forName(ConfigurationKeys.DEFAULT_CHARSET_ENCODING)), ',', '\"');
```

Comment:
Can you explain here why you need to change the instructor InputStreamCSVReader(String input), instead of using InputStreamCSVReader(Reader input, char customizedSeparator) directly in your codes? Because this changes the original purpose of InputStreamCSVReader(String input).